rpi_metar/airports.py

```diff
@@ -85,7 +85,7 @@ def process_metar(self, metars):
             return
 
         # Thunderstorms
-        self.thunderstorms = any(word in metar['raw_text'] for word in ['TSRA', 'VCTS']) and self.category != wx.FlightCategory.OFF
+        self.thunderstorms = any(word in metar['raw_text'] for word in ['TSRA', 'VCTS', 'TS ']) and self.category != wx.FlightCategory.OFF
```

Comment:
```suggestion
        self.thunderstorms = bool(re.search(r'\w{4,5}.*(TS).*?(?=RMK)', metar['raw_text']) and self.category != wx.FlightCategory.OFF)
```

Reply:
I am by no means an expert in regex.  It would, however be a good approach to the issue. 

The regex here seemed good. https://github.com/ScottSturdivant/rpi_metar/pull/9#issuecomment-498330218 

I am not sure the new one is perfect

Reply:
If you want to update your PR to do this regex approach with the regex from #9, I'm OK with that and will accept it.